bin/wmagent-couchapp-init

```diff
@@ -18,6 +18,19 @@ from WMCore.Configuration import loadConfigurationFile
 from WMCore.Lexicon import splitCouchServiceURL
 from WMCore.WMBase import getWMBASE
 
+parser = argparse.ArgumentParser()
+parser.add_argument("--skip-cron", dest = "cron",
+                    default = True, action = "store_false",
+                    help = "Do not install maintenance cron jobs")
+parser.add_argument("--only-cron", dest = "pushApps",
+                    default = True, action = "store_false",
+                    help = "Install only maintenance cron jobs")
+options = parser.parse_args()
+
+if not options.cron and not options.pushApps:
```

Comment:
Just a minor comment, this could have been accomplished with `add_mutually_exclusive_group()`

Reply:
nice idea @amaltaro  thanks !